# Hackbio Internship Stage 0: Learning Track 2 (Biomarker Discovery)

## TP53 - Guardian of the Genome

TP53, or tumour protein 53, is a crucial tumour suppressor gene that plays a key role in maintaining the stability of the genome and preventing cancer formation in vertebrates. Studies have shown that mutation within this gene is associated with tumorigenesis, ultimately leading to cancer. Upon activation, the p53 protein typically hinders tumour growth or triggers apoptosis by regulating the transcription of nearly 500 genes, thereby controlling diverse cellular processes (Guo et al., 2000; Chen et al., 2022). Recent studies have reported that mutp53 strongly correlates with advanced malignancies and poor prognosis, making it an ideal prognostic biomarker and drug target (Chen et al., 2022). Almost 80% of the mutations in p53 are missense mutations/point mutations in the DNA binding domain of the protein, which, in turn, provides a gain-of-function (GOF) to the p53 gene and aids in tumour growth (Kukurba and Montgomery, 2015).

![image](https://github.com/user-attachments/assets/f2b89220-a8ee-4f2a-8575-f04475a3ce84)

*Structure of TP53 protein from PDB (Vogelstein, Lane and Levine, 2000) [View 3D structure of TP53] (https://www.ncbi.nlm.nih.gov/Structure/icn3d/full.html?mmdbafid=Q12888&bu=1)*


Recent advancements in RNA sequencing and spatial transcriptomics suggest that mutp53 can be exploited as a potential prognostic biomarker as it is prevalent in almost 50% of cancers, including ovarian, breast, colorectal, and lung cancer (Chen et al., 2022). RNA-seq data is extensively utilised in analysing genes associated with p53 and biological pathways related to tumour induction and progression. The RNA-seq data from cell lines and patient samples in TCGA (https\://portal.gdc.cancer.gov/) can be utilised to analyse the expression of TP53 splice variants (Mehta et al., 2016). Sánchez et al. (2014) demonstrated that RNA-seq data of p53 can be integrated with ChIP-seq to identify the changes in gene expression patterns based on the binding of the p53 protein, identifying 27 lnc-RNAs associated with p53 in response to DNA damage. Differential gene expression analysis reveals the complex molecular mechanisms of cancer and facilitates the discovery of potential biomarkers (M et al., 2020). Various unannotated transcripts and novel lncRNAs can be identified by analysing the gene expression data of various mutp53 cells, which could serve as biomarkers in the future. Yang et al. (2021) have reported that TP53 neoantigens can be utilised as a prognostic biomarker as it mediates anti-tumour activity in HCC (Hepatocellular carcinoma) or a potential treatment option for cancer immunotherapy. Overall, these studies suggest that mutp53 is an excellent choice for a multiplexable biomarker that can be used for identifying mutants from circulating tumor DNA by analyzing blood samples of patients, potentially revolutionizing cancer diagnosis and monitoring.

## References
Aubrey, B.J. et al. (2018) ‘How does p53 induce apoptosis and how does this relate to p53-mediated tumour suppression?’, Cell Death & Differentiation, 25(1), pp. 104–113. Available at: https://doi.org/10.1038/cdd.2017.169.

Chen, X. et al. (2022) ‘Mutant p53 in cancer: from molecular mechanism to therapeutic modulation’, Cell Death & Disease, 13(11), p. 974. Available at: https://doi.org/10.1038/s41419-022-05408-1.

Guo, A. et al. (2000) ‘The function of PML in p53-dependent apoptosis’, Nature Cell Biology, 2(10), pp. 730–736. Available at: https://doi.org/10.1038/35036365.

Hong, M. et al. (2020) ‘RNA sequencing: new technologies and applications in cancer research’, Journal of Hematology & Oncology, 13, p. 166. Available at: https://doi.org/10.1186/s13045-020-01005-x.

Kukurba, K.R. and Montgomery, S.B. (2015) ‘Rna sequencing and analysis’, Cold Spring Harbor Protocols, 2015(11), p. pdb.top084970. Available at: https://doi.org/10.1101/pdb.top084970.

M, G. et al. (2020) ‘High-throughput approaches for precision medicine in high-grade serous ovarian cancer’, Journal of hematology & oncology, 13(1). Available at: https://doi.org/10.1186/s13045-020-00971-6.

Mehta, S. et al. (2016) ‘A study of tp53 rna splicing illustrates pitfalls of rna-seq methodology’, Cancer Research, 76(24), pp. 7151–7159. Available at: https://doi.org/10.1158/0008-5472.CAN-16-1624.

Sánchez, Y. et al. (2014) ‘Genome-wide analysis of the human p53 transcriptional network unveils a lncRNA tumour suppressor signature’, Nature Communications, 5(1), p. 5812. Available at: https://doi.org/10.1038/ncomms6812.

Vogelstein, B., Lane, D. and Levine, A.J. (2000) ‘Surfing the p53 network’, Nature, 408(6810), pp. 307–310. Available at: https://doi.org/10.1038/35042675.

Yang, H. et al. (2021) ‘Unique TP53 neoantigen and the immune microenvironment in long-term survivors of Hepatocellular carcinoma’, Cancer immunology, immunotherapy: CII, 70(3), pp. 667–677. Available at: https://doi.org/10.1007/s00262-020-02711-8.
