# RESEARCH PAPER SUMMARY

** Authors: Shola Paul Emiade (@Shawlyem), Abilashni Arthiswaran (@Abilashni83) and Gobikrishnan Subramaniam (@gobi3141)**

## (Comprehensive molecular profiling of lung adenocarcinoma) [https://www.nature.com/articles/nature13385]

## Introduction
According to GLOBOCAN 2022 estimates from the Global Cancer Observatory,  lung adenocarcinoma was found to be the leading cause of cancer mortality worldwide with over 1.8 million deaths. Recent advancements in next-generation sequencing (NGS) have significantly transformed the landscape of tumour diagnosis and treatment by enabling the utilisation of molecular profiling to improve diagnosis and optimise treatment plans. This study focuses on identifying mutations that can be utilised for targeted therapies involving clinically actionable genes, and understanding the role of key factors driving lung adenocarcinoma progression and provides new paradigms for developing new targeted therapies and personalised treatments (Collisson et al., 2014, Ding et al., 2008; Imielinski et al., 2012). 

## Study design and methodology
230 samples, were collected from adenocarcinoma patients with all major histologic forms of the cancer represented. DNA and RNA were extracted using the Allprep kit and whole-exome sequencing (WES) was performed on tumour and germline DNA. Consensus clustering approaches were used to analyse mRNA, miRNA and methylation subtypes (Collisson et al., 2014).

## Key findings
Higher frequency of somatic mutations were observed, with a mean of 8.9 mutations per megabase and among them 18 genes were statistically significantly (RIT1 activating mutations and MGA loss-of-function mutations).
Mutations in EGFR were more frequent in female patients, while RBM10 mutations were more common in males. 
KRAS mutations were more prevalent in smokers, while EGFR mutations were enriched in non-smokers, with distinct mutation patterns observed between these groups. 
Aberrations in NF1, MET, ERBB2 and RIT1 occurred in 13% of cases and were enriched in samples lacking an activated oncogene, suggesting a potential driver role.
DNA and RNA sequencing revealed splicing alterations driven by somatic genomic changes, including MET exon 14 skipping in 4% of cases. 
Proteomic analysis showed that MAPK and PI(3)K pathway activation was not fully explained by known mutations, suggesting additional unidentified mechanisms of pathway activation. 
Key driver mutations were identified in the RTK/RAS/RAF pathway, including KRAS, EGFR, and BRAF, increasing the detection of pathway activation from 62% to 76% and expanding the range of targetable mutations for treatment.
Integrative analysis identified 6 distinct molecular subtypes of lung adenocarcinoma associated with distinct genomic alterations.

## Conclusion
The molecular diversity in lung adenocarcinoma emphasises the need for more refined classifications to advance precision medicine. Further investigation should focus on molecular sub-classifications for precision medicine, with particular attention to epigenetic regulators,  mechanisms of pathway activation at the protein level and RNA splicing are crucial for improving treatment effectiveness. This research reinforces the importance of exploring molecular diversity of lung adenocarcinoma to identify novel biomarkers for early detection and better prognosis as well as aid in the development of personalised cancer therapeutics.

## References
Bergethon, K. et al. (2012) ‘ros1 rearrangements define a unique molecular class of lung cancers’, Journal of Clinical Oncology, 30(8), pp. 863–870. Available at: https://doi.org/10.1200/JCO.2011.35.6345.

Collisson, E.A. et al. (2014) ‘Comprehensive molecular profiling of lung adenocarcinoma’, Nature, 511(7511), pp. 543–550. Available at: https://doi.org/10.1038/nature13385.

Ding, L. et al. (2008) ‘Somatic mutations affect key pathways in lung adenocarcinoma’, Nature, 455(7216), pp. 1069–1075. Available at: https://doi.org/10.1038/nature07423.

Drilon, A. et al. (2013) ‘Response to cabozantinib in patients with ret fusion-positive lung adenocarcinomas’, Cancer Discovery, 3(6), pp. 630–635. Available at: https://doi.org/10.1158/2159-8290.CD-13-0035.

Ferlay J, Ervik M, Lam F, Laversanne M, Colombet M, Mery L, Piñeros M, Znaor A, Soerjomataram I, Bray F (2024).
Global Cancer Observatory: Cancer Today. Lyon, France: International Agency for Research on Cancer. Available
from: https://gco.iarc.who.int/today, accessed [12 Sep 2024].

Imielinski, M. et al. (2012) ‘Mapping the hallmarks of lung adenocarcinoma with massively parallel sequencing’, Cell, 150(6), pp. 1107–1120. Available at: https://doi.org/10.1016/j.cell.2012.08.029.

Kwak, E.L. et al. (2010) ‘Anaplastic lymphoma kinase inhibition in non–small-cell lung cancer’, New England Journal of Medicine, 363(18), pp. 1693–1703. Available at: https://doi.org/10.1056/NEJMoa1006448.

Paez, J.G. et al. (2004) ‘egfr mutations in lung cancer: correlation with clinical response to gefitinib therapy’, Science, 304(5676), pp. 1497–1500. Available at: https://doi.org/10.1126/science.1099314.

Stephens, P. et al. (2004) ‘Intragenic ERBB2 kinase mutations in tumours’, Nature, 431(7008), pp. 525–526. Available at: https://doi.org/10.1038/431525b.

The Cancer Genome Atlas Research Network (2012) ‘Comprehensive genomic characterization of squamous cell lung cancers’, Nature, 489(7417), pp. 519–525. Available at: https://doi.org/10.1038/nature11404.
